src/Akka.Persistence.MongoDb/Akka.Persistence.MongoDb.csproj

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetStandardLibVersion)</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
```

Comment:
Added this in order to support `using` statements, which I ended up not even needing in most cases.